ckanext/benap/i18n/ckanext-benap.pot

```diff
@@ -33,6 +33,14 @@ msgstr ""
 msgid "URL {url} has to start with https://"
 msgstr ""
 
+#: ckanext/benap/validators.py:92
+msgid "Modification must done by a system administrator"
+msgstr ""
+
+#: ckanext/benap/validators.py:96
+msgid "Logged in one must be"
+msgstr ""
+
 #: ckanext/benap/templates/base.html:107
 msgid ""
```

Comment:
geen msgid hier, ok?

Reply:
Pot files zijn the catalog template om de vertalingen op te bouwen. Waardes worden ingevuld wanneer er van de template een specifieke taal wordt afgeleid

Reply:
Ok, dan is het goed zo! Was nog niet mee met de logica ;)